src/style.css

```diff
@@ -2,0 +1,63 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-weight: bold;
+}
+
+html {
+  margin: 0;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, Arial, sans-serif;
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search-input::placeholder {
+  color: #3d4e61;
+}
+
+.search-input--big {
+  height: 70px;
+  line-height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background-size: 19px;
+  background-position: 26px center;
+}
+
+.search-input--small {
+  height: 42px;
+  line-height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-size: 11px;
+  background-position: 13px center;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-input:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+  font-weight: 900;
```

Comment:
The `font-weight` property here is set to `900`, which might not be supported by the Avenir font as per the defined `@font-face` rules. Ensure there's a corresponding `@font-face` rule that supports the `900` font-weight or adjust this value accordingly.